app/src/main/java/org/simple/clinic/patient/businessid/BusinessIdMetaData.kt

```diff
@@ -1,10 +1,12 @@
 package org.simple.clinic.patient.businessid
```

Comment:
Should these changes be part of this PR?

Reply:
Yup, the moshi dependency requires this.